multimedia/VLC2/Portfile

```diff
@@ -299,8 +307,8 @@ if {(${subport} eq ${name}) || (${subport} eq "lib${name}")} {
         configure.args-replace  --disable-shout --enable-shout
     }
 
-    variant smb description {Enable Samba 3 input support} {
-        depends_lib-append      port:samba3
+    variant smb description {Enable Samba 3+ input support} {
+        depends_lib-append      bin:smbclient:samba3
```

Comment:
What's going on here? Why do you want to allow an `smbclient` installed outside of MacPorts to satisfy this dependency?

Reply:
Erm, yes, that should be a path-style depspec. I've been working on a port:samba4 implementation (which doesn't yet work 100% on Mac) hence the need for a flexible dependency